### PR TITLE
Migration to DMN v1.3

### PR DESCRIPTION
**Pull request by tarilabs**
currently a work-in-progress PR to migrate DMN TCK to DMN v1.3

I have migrated namespaces accordingly to OMG RTF github repo and documents.
I have included XSD from the same.
I have aligned dmn-tck-validation Runner for the new DMN v1.3 XSD.

I want to review a couple of test case for which I raised an RTF issue that was solved with DMNv1.3; besides of that, I believe this should be enough to migrate the test cases to DMNv1.3, website alignment will need to follow.

This PR leaves the default of allow edits, and in case somebody want to append other changes feel free to do so also on my fork+branch.

### CODE REVIEW

**Comment by StrayAlien** [comment]
Hi @tarilabs - is there a draft of 1.3 available as yet? Or at least somewhere that says what the changes are?

**Comment by tarilabs** [comment]
Hi @StrayAlien Happy New 2020!
This article summarized it better than I can do the current status: https://dmcommunity.org/2019/12/10/dmn-1-3-approved

**Comment by StrayAlien** [comment]
Thanks @tarilabs - the temporal stuff looks very interesting indeed - is that all coming in 1.3?  That is a *lot* of tests! :-) .... actually the largest gap in the TCK (IMO) is the operations between types (especially dates etc).  We have very little specific coverage there.

I'm afraid I've not to $$ to be one of the cool kids in the OMG club so all I see is the temporal article.  Any other stuff to note?

Greg.

**Comment by tarilabs** [comment]
> I'm afraid I've not to $$ to be one of the cool kids in the OMG club so all I see is the temporal article. Any other stuff to note?

fwiw, OMG is a not-for-profit consortium organisation, so I believe the fees are a way to sustain the organisation efforts, with their merits or limitations; referring it as a "_club of cool kids_" imho is a bit stretched 😉 

I believe the other related stuff to note was the general discussion about DMN which happened at DecisionCamp 2019, I believe these notes could be somehow of interest?: https://dmcommunity.org/2019/10/05/notes-from-decisioncamp-2019

Hope this helps.

**Comment by StrayAlien** [comment]
Thanks Matteo, appreciated.  Thanks for the link.  A very interesting summary.  Though I guess I am going to have to wait to see what is in the 1.3 spec.

Re being cool - that was a joke - not serious at all.  But, I have sure learned that decision-related topics are not a great conversation-starter! I find it all very fascinating but, alas, many do not. Almost a sure fire guaranteed to bore practically anyone.  Except us lot that is.  :-)

As always, thanks again.

Greg.

**Comment by tarilabs** [comment]
As discussed at TCK meeting 2020-04-24, this PR aligns the test files to the file format of DMNv1.3, and align those test results which are necessarily changed with clarifications present in DMNv1.3. Will proceed to merge.

### Alignment to DMNv1.3 XSDs, namespaces, etc.
Align file format, namespaces etc to DMNv1.3
Align TCK Validation runner to DMNv1.3 files
Notice this does indeed include the latests changed voted in the last ballots of the RTF.
commits ef8fb2e 6c44714 23c7ad8

### DMNv1.3 DMN13-163 Decision Service output value in the case of single output Decision
In short: a Decision Service of a single decision, is not a context of a single context-entry of said decision name and its value, but coerce to the singleton decision value. This is now clarified to the original intent according to the chair, and the same for externally executed PMML.
commit 128d9ae

### DMNv1.3 DMN13-139/DMN13-182 Align abs(n) to built-in function param names 
commit ab79722

### DMNv1.3 DMN13-127/DMN13-165 Align built-in string function
In short: it was made explicit String length calculation is based on Unicode codepoint
commit b1116ff
